src/js/components/AppConfigEditFormComponent.jsx

```diff
@@ -358,6 +359,15 @@ var AppConfigEditFormComponent = React.createClass({
                   <input min="0" step="any" type="number"/>
                 </FormGroupComponent>
               </div>
+              <div className="col-sm-3">
+                <FormGroupComponent fieldId="networkBandwidth"
+                  label="Network (Mbps)"
+                  errorMessage={this.getErrorMessage("networkBandwidth")}
+                  value={state.fields.networkBandwidth}
+                  onChange={this.handleFieldUpdate}>
+                  <input min="0" step="any" type="number"/>
```

Comment:
Since we ask the users to set a non zero value for Mesos to take it into account I would put 1 Mbps as the minimum value and 10Mbps as the default value. What do you think?

Reply:
Well, I've just seen you put the default to 50, I'm ok with that.